src/components/Property/Contacts/TenantsContactTable/TenantContact.js

```diff
@@ -0,0 +1,50 @@
+import { useMemo } from 'react'
+import TenantHeading from './TenantHeading'
+import TenantPhoneNumber from './TenantPhoneNumber'
+
+const TenanctContact = ({ tenant, reloadContacts }) => {
+  const { phoneNumbers } = tenant
+
+  const sortedPhoneNumbers = useMemo(
+    () =>
+      phoneNumbers
+        .slice() // create a shallow copy to avoid mutating the original array
```

Comment:
Using a destructure e.g. `numbersCopy = [...numbers];` would avoid you having to explain the slice